keccak256/src/permutation/rho.rs

```diff
@@ -14,7 +14,7 @@ use std::convert::TryInto;
 #[derive(Debug, Clone)]
 pub struct RhoConfig<F> {
     state: [Column<Advice>; 25],
```

Comment:
These 25 columns of state are unused now.
I think it's because we disabled the dead code check somewhere.
We can replace the `state: [Column<Advice>; 25]` with `lane: Column<Advice>` and do a side effect for enabling column copy.

Reply:
I'll address all of the unused code in a follow-up PR. 

Want to try to keep this one just about the performance improvements.

Reply:
Filed https://github.com/appliedzkp/zkevm-circuits/issues/512